content/en/docs/setup/getting-started/index.md

```diff
@@ -72,142 +56,341 @@ Download the Istio release which includes installation files, samples, and the
-    istiod                      ClusterIP      172.21.137.226   <none>           15012/TCP,443/TCP                                                                                                            28s
-    jaeger-agent                ClusterIP      None             <none>           5775/UDP,6831/UDP,6832/UDP                                                                                                   12s
-    jaeger-collector            ClusterIP      172.21.227.125   <none>           14267/TCP,14268/TCP,14250/TCP                                                                                                12s
-    jaeger-collector-headless   ClusterIP      None             <none>           14250/TCP                                                                                                                    12s
-    jaeger-query                ClusterIP      172.21.94.208    <none>           16686/TCP                                                                                                                    12s
-    kiali                       ClusterIP      172.21.192.82    <none>           20001/TCP                                                                                                                    11s
-    prometheus                  ClusterIP      172.21.30.218    <none>           9090/TCP                                                                                                                     11s
-    tracing                     ClusterIP      172.21.126.166   <none>           80/TCP                                                                                                                       11s
-    zipkin                      ClusterIP      172.21.122.59    <none>           9411/TCP                                                                                                                     11s
+    $ kubectl get pods
+    NAME                              READY   STATUS            RESTARTS   AGE
+    details-v1-78d78fbddf-tj56d       0/2     PodInitializing   0          2m30s
+    productpage-v1-85b9bf9cd7-zg7tr   0/2     PodInitializing   0          2m29s
+    ratings-v1-6c9dbf6b45-5djtx       0/2     PodInitializing   0          2m29s
+    reviews-v1-564b97f875-dzdt5       0/2     PodInitializing   0          2m30s
+    reviews-v2-568c7c9d8f-p5wrj       1/2     Running           0          2m29s
+    reviews-v3-67b4988599-7nhwz       0/2     PodInitializing   0          2m29s
     {{< /text >}}
 
     {{< tip >}}
-    If your cluster runs in an environment that does not
-    support an external load balancer (e.g., minikube), the
-    `EXTERNAL-IP` of `istio-ingressgateway` will display
-    `<pending>`. To access the gateway, use the service's
-    `NodePort`, or use port-forwarding instead.
+    Re-run the previous command and wait until all pods report READY 2 / 2 and
```

Comment:
@howardjohn had a good example in another repo. This might work instead:

kubectl wait --for=condition=Available deployment --all --timeout=120s -n default